src/components/menu.component.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+import React from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+const {
+  Component,
```

Comment:
These can be put on the same line as the import React statement.

e.g. 

import React, { Component, NavigatorIOS } from ' react-native'